src/toplevel_generators.jl

```diff
@@ -158,13 +164,15 @@ function generate_wrapper_load(src_name, pkg_uuid, __source__)
                 end
 
                 # From the available options, choose the best wrapper script
-                select_platform(valid_wrappers)
+                platform = augment_platform(HostPlatform())
```

Comment:
Let's do our best to only invoke `augment_platform()` once; it's supposed to be lightweight, but in the event that it's not, let's minimize the amount of work we do.